api/query/cache/index/aggregator.go

```diff
@@ -32,7 +33,26 @@ func (a *indexAggregator) Add(t TestID) error {
 			return err
 		}
 
-		r = query.SearchResult{Test: name, LegacyStatus: nil}
+		r = query.SearchResult{
+			Test:         name,
+			LegacyStatus: nil,
+			Interop:      nil,
+		}
+	}
+
+	if a.interopFormat {
+		if r.Interop == nil {
+			r.Interop = make([]int, len(a.rus)+1)
+		}
+		passing := 0
+		for _, ru := range a.rus {
+			res := shared.TestStatus(a.runResults[ru].GetResult(t))
+			// Only include tests with non-UNKNOWN status for this run's total.
```

Comment:
The commend doesn't match what the code seems to be doing, as there are many statuses other than PASS or OK. Which is correct?

Reply:
The code :)